vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -1654,7 +1654,13 @@ private void fetchWorkflowVersion(HttpServerExchange exchange) {
                       .on(WORKFLOW_VERSION.WORKFLOW_DEFINITION.eq(WORKFLOW_DEFINITION.ID)))
               .where(WORKFLOW_VERSION.NAME.eq(name).and(WORKFLOW_VERSION.VERSION.eq(version)))
               .fetchOptional()
-              .map(r -> r.value1().data());
+              .map(
+                  r -> {
+                    if (r.value1() != null) {
+                      return r.value1().data();
+                    }
+                    return null;
+                  });
```

Comment:
This can also be represented as
```
  .fetchOptional()
  .value1())  // note the extra close-parentheses here
.map(r -> r.data())
.orElseGet(() -> "");
```
as used elsewhere in the file: https://github.com/oicr-gsi/vidarr/blob/38ab38e81a745529923594765f0ca36601943c85/vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java#L2661

Reply:
^ I ended up trying out the representation I suggested and it does not work at all, so I retract the suggestion!